plugins/module_utils/_crypto/module_backends/csr.py

```diff
@@ -185,6 +264,10 @@ def __init__(self, *, module: AnsibleModule) -> None:
         self.crl_distribution_points: (
             list[cryptography.x509.DistributionPoint] | None
         ) = None
+        self.custom_extensions: (
+            list[tuple[cryptography.x509.UnrecognizedExtension, bool]] | None
+        ) = None
```

Comment:
I would remove the `| None` and set it to `[]`. There's no need to distinguish between `None` and `[]`.